process/economicMetrics.go

```diff
@@ -29,7 +28,7 @@ func (nsp *NodeStatusProcessor) getEconomicsDataMetricsFromApi() (*data.GenericA
 }
 
 func (nsp *NodeStatusProcessor) getEconomicsDataMetrics(observers []*data.NodeData) (*data.GenericAPIResponse, error) {
-	var responseNetworkMetrics data.GenericAPIResponse
+	responseNetworkMetrics := &data.GenericAPIResponse{}
 	for _, observer := range observers {
 
 		_, err := nsp.proc.CallGetRestEndPoint(observer.Address, EconomicsDataPath, &responseNetworkMetrics)
```

Comment:
double `&` -> `responseNetworkMetrics := &data.GenericAPIResponse{}` and `&responseNetworkMetrics`- does it work as expected?